docs/css/style.css

```
@@ -74,28 +74,39 @@ nav.badges a + a {
   margin-left: 3px;
 }
 
-.image-list {
+ul.image-list {
   overflow: hidden;
   text-align: center;
+  list-style: none;
+  column-count: 1;
+  padding: 0;
+  margin: 0;
 }
 
-.image-list a {
+ul.image-list li {
+  border-bottom: none;
   display: inline-block;
-  margin: 6px;
+  margin: 0 4px 0 4px;
+  max-height: 64px;
+  padding: 0;
 }
 
-.image-list a img {
+ul.image-list li a {
+  display: inline-block;
+}
+
+ul.image-list li img {
+  margin: 0;
+  padding: 0;
   display: block;
-  height: 64px;
+  max-height: 64px;
 }
 
-.faded-images {
-  background-color: #ddd;
-  border: 1px solid;
-  border-color: #ddd #ddd #ccc;
-  border-radius: 3px;
-  padding: 1em;
-  box-shadow: inset 0 0 10px #ccc;
+ul#backers.image-list li img {
+  width: 32px;
+  height: 32px;
+  /* clip images to 32x32 */
+  clip-path: inset(0, 0, 32px, 32px);
```

Backers have fixed dimensions, so the images take up space. That means we can put a background color on them and let them display early as grey squares

```suggestion
  clip-path: inset(0, 0, 32px, 32px);
  background-color: #969ba2
```

`#969ba2` is the background color opencollective uses on their missing avatar images, so I'm attempting to optimize for the least change when images load